src/software/thunderscope/colors.py

```diff
@@ -2,17 +2,19 @@
 
 
 class Colors(object):
+
     FIELD_COLOR = "w"
     FIELD_LINE_COLOR = "w"
 
     BLUE_ROBOT = QtGui.QColor(255, 100, 0, 255)
     BALL_COLOR = QtGui.QColor(255, 100, 0, 255)
+    SIM_BALL_COLOR = QtGui.QColor(255, 100, 0, 150)
     YELLOW_ROBOT_COLOR = QtGui.QColor(255, 255, 0, 255)
     BLUE_ROBOT_COLOR = QtGui.QColor(0, 75, 255, 255)
 
     ROBOT_SPEED_SLOW_COLOR = "black"
-    NAVIGATOR_PATH_COLOR = "b"
-    NAVIGATOR_OBSTACLE_COLOR = "r"
+    NAVIGATOR_PATH_COLOR = "green"
+    NAVIGATOR_OBSTACLE_COLOR = "orange"
```

Comment:
can we use a colour palate instead of these bright colours?

Reply:
yeah we can, I picked what I thought looked good

open to suggestions

Reply:
it'd be interesting to parameterize it, like my_color_palette[0], my_color_palette[1], etc.